lib/systems/architectures.nix

```diff
@@ -26,6 +26,7 @@ rec {
     cooperlake     = [ "sse3" "ssse3" "sse4_1" "sse4_2"         "aes" "avx" "avx2" "avx512" "fma"        ];
     tigerlake      = [ "sse3" "ssse3" "sse4_1" "sse4_2"         "aes" "avx" "avx2" "avx512" "fma"        ];
     alderlake      = [ "sse3" "ssse3" "sse4_1" "sse4_2"         "aes" "avx" "avx2"          "fma"        ];
+    sapphirerapids = [ "sse3" "ssse3" "sse4_1" "sse4_2"         "aes" "avx" "avx2" "avx512" "fma"        ];
```

Comment:
Will check again for sse4a

Reply:
Ok. Looks correct to me. This is /proc/cpuinfo: https://gist.github.com/Mic92/a04ce8f76efded149c19351d649670d7